frontend/types/api/validator_dashboard.ts

```diff
@@ -54,7 +54,8 @@ export interface VDBSummaryTableRow {
 export type GetValidatorDashboardSummaryResponse = ApiPagingResponse<VDBSummaryTableRow>;
 export interface VDBGroupSummaryColumnItem {
   status_count: StatusCount;
-  validators?: number /* uint64 */[];
+  validators?: number /* uint64 */[]; // fill with up to 3 validator indexes
+  validator_count: number /* uint64 */; // number of distinct validators
```

Comment:
This will contain the count for sync and slash validators, within the `sync` and `slashings` field of a `VDBGroupSummarySyncCount`.